scripts/scil_compute_metrics_stats_in_ROI.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Compute the statistics (mean, stddev) of scalar maps, which can represent
+diffusion metrics, in a ROI.
+
+The mask can either be a binary mask, or a weighting mask. If a weighting mask
+should either contain floats between 0 and 1, or should be normalized with
+--normalize_weights.
+
+IMPORTANT: if the mask contains weights (and not 0 and 1 exclusively), the
+standard deviation will also be weighted.
+"""
+
+import argparse
+import nibabel as nib
+import numpy as np
+import os
+import json
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_json_args,
+                             assert_inputs_exist)
+from scilpy.utils.filenames import split_name_with_nii
+from scilpy.utils.metrics_tools import get_roi_metrics_stats
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_mask',
+                   help='Mask volume file name, formatted in any nibabel ' +
```

Comment:
We usually use filename I think. Also I think usually we don't put the '+' at the end.